extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/GeneratedStaticResourcesBuildItem.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.devui.deployment;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.vertx.http.deployment.spi.GeneratedStaticResourceBuildItem;
+
+/**
+ * This build items aims to be a collection of all {@link GeneratedStaticResourceBuildItem} produced by extensions.
+ */
+public final class GeneratedStaticResourcesBuildItem extends SimpleBuildItem {
```

Comment:
What's the purpose of this one?
Asking because it seems to be a bit redundant with `GeneratedStaticResourceBuildItem`. Except if it aggregates things from other build items  but it doesn't look obvious from the javadoc?

I don't think we have this sort of things elsewhere so except if it has a value I missed, I would drop it.

Reply:
> What's the purpose of this one?

This means be a collection of `GeneratedStaticresourceBuildItem`, I removed it and generated another `@BuildStep` with `@Record`.